recipe/meta.yaml

```diff
@@ -1,57 +1,100 @@
-{% set name = "msgspec" %}
+{% set name = "msgspec-packages" %}
```

Comment:
```suggestion
{% set name = "msgspec" %}
```

Reply:
we should align with PyPi package names.